### PR TITLE
Improve sample for boundingbox

### PR DESCRIPTION
**Pull request by maririos**
Outcome from https://github.com/Azure/azure-sdk-for-net/issues/12721.
Because Bounding box is everywhere, I decided to create an extra sample file with it and its description

### CODE REVIEW

File: sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample8_BoundingBox.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+using Azure.AI.FormRecognizer.Tests;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.FormRecognizer.Samples
+{
+    public partial class FormRecognizerSamples : SamplesBase<FormRecognizerTestEnvironment>
+    {
+        /// <summary>
+        /// This sample illustrates how to consume Bounding Box type data that is available across the Form Recognizer library.
```
Comment (annelo-msft) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample8_BoundingBox.cs:17]: Is it worth including in the introductory comment a description of what a Bounding Box is?  Some people don't know this when they first come to the library.  Also worth noting units and where they're found.

Comment (maririos) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample8_BoundingBox.cs:17]: hmm great point. Will add

File: sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample8_BoundingBox.cs [hunk 2]
```diff
@@ -14,7 +14,12 @@ namespace Azure.AI.FormRecognizer.Samples
     public partial class FormRecognizerSamples : SamplesBase<FormRecognizerTestEnvironment>
     {
         /// <summary>
-        /// This sample illustrates how to consume Bounding Box type data that is available across the Form Recognizer library.
+        /// This sample illustrates how to consume a Bounding Box type data that is available across the Form Recognizer library.
+        /// A bounding box is a sequence of four <see cref="PointF"/> representing a quadrilateral that outlines
+        /// the text of an element in a recognized form.
```
Comment (annelo-msft) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample8_BoundingBox.cs:19]: nit: in v2.1 there may be non-textual elements - maybe say and element, such as a line or a word, ... ?

File: sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample8_BoundingBox.cs [hunk 3]
```diff
@@ -14,7 +14,12 @@ namespace Azure.AI.FormRecognizer.Samples
...
+        /// Coordinates are specified relative to the top-left of the original image,
+        /// and points are ordered clockwise from the top-left corner relative to the text orientation.
+        /// Units are in pixels for images and inches for PDF.
```
Comment (annelo-msft) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample8_BoundingBox.cs:22]: Can we instead say where the unit can be found?  We have this in an enum.

**Review by annelo-msft — APPROVED** [review]

File: sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample8_BoundingBox.cs (hunk 2 above)
Comment (maririos) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample8_BoundingBox.cs:19]: thank you! I was not aware

**Pull request merged**